CMake/ctest_script.cmake

```diff
@@ -40,18 +40,20 @@ ELSEIF( ${HOSTNAME} MATCHES "cori" )
     SET( N_PROCS 32)
 ELSEIF( ${HOSTNAME} MATCHES "eos" )
 # Setup for eos.ccs.ornl.gov Cray XC30 Intel E5-2670 Aries interconnect.
+# N_NPROCS and N_PROCS_BUILD should be set to respect current OLCF limits
     SET( CC ${CMAKE_C_COMPILER} )
     SET( CXX ${CMAKE_CXX_COMPILER} )
     SET( CTEST_CMAKE_GENERATOR "Unix Makefiles")
     SET( CTEST_SITE "eos.ccs.ornl.gov" )
-    SET( N_PROCS 16)
+    SET( N_PROCS_BUILD 8 )
 ELSEIF( ${HOSTNAME} MATCHES "titan" )
```

Comment:
In principle, this can be removed since you added the environment variable.

Reply:
Yes it could, but we are not going to. OLCF has process limits that cause weird build errors if this is not set. If someone wants to experiment, the environment variable takes precedence.

Reply:
fair enough.

Reply:
Please merge - I need this merged before I can finish testing the job scripts.

Reply:
Could you fix the previous two places. Add an else defining N_PROCS_BUILD and then always set make with N_PROCS_BUILD.